aws/resource_aws_api_gateway_integration_test.go

```diff
@@ -272,11 +272,10 @@ func TestAccAWSAPIGatewayIntegration_integrationType(t *testing.T) {
 				),
 			},
 			{
-				Config: testAccAWSAPIGatewayIntegrationConfig_IntegrationTypeInternet(rName),
+				Config: testAccAWSAPIGatewayIntegrationConfig_IntegrationTLSConfig(rName),
```

Comment:
Will move this to its own acceptance test and make the `insecure_skip_verification` a boolean function parameter so we can easily verify updates between true/false.